python/DEVELOPER.md

```diff
@@ -33,6 +33,7 @@ source "$HOME/.cargo/env"
 rustc --version
 # Install protobuf compiler
 PB_REL="https://github.com/protocolbuffers/protobuf/releases"
+# For other arch the signature of the should be protoc-<version>-<os>-<arch>.zip, e.g. protoc-3.20.3-linux-aarch_64.zip for ARM64.
```

Comment:
For other arch -> other than what?  

 signature of the  -> please fix

Please make this comment clearer